pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2313,4 +2312,15 @@ self: super: {
   # multiple bounds too strict
   co-log-polysemy = doJailbreak super.co-log-polysemy;
   co-log-polysemy-formatting = doJailbreak super.co-log-polysemy-formatting;
+
+  # 2022-12-02: Needs newer postgrest package
+  postgrest = dontCheck (doJailbreak (overrideCabal (drv: rec {
```

This ensures the build can't be broken by hackage revisions.

```suggestion
  postgrest = dontCheck (doJailbreak (overrideSrc (drv: rec {
```

---

Done.